src/meta/play-meta.js

```diff
@@ -27,7 +27,8 @@ import {
   NavBar,
```

Comment:
This play was created with the old flow. This file shouldn't be committed anymore